.github/workflows/integration-tdx.yml

```diff
@@ -57,48 +57,48 @@ jobs:
 
       - name: Run Tests - Boot Kernel
         run: |
-          bash sh_script/integration_tdx.sh -c 1 -m 1G -f target/x86_64-unknown-uefi/release/final-boot-kernel.bin
-          bash sh_script/integration_tdx.sh -c 1 -m 2G -f target/x86_64-unknown-uefi/release/final-boot-kernel.bin
-          bash sh_script/integration_tdx.sh -c 2 -m 4G -f target/x86_64-unknown-uefi/release/final-boot-kernel.bin
-          bash sh_script/integration_tdx.sh -c 4 -m 8G -f target/x86_64-unknown-uefi/release/final-boot-kernel.bin
-          bash sh_script/integration_tdx.sh -c 8 -m 16G -f target/x86_64-unknown-uefi/release/final-boot-kernel.bin
+          bash sh_script/integration_tdx.sh -c 1 -m 1G -f target/release/final-boot-kernel.bin
+          bash sh_script/integration_tdx.sh -c 1 -m 2G -f target/release/final-boot-kernel.bin
+          bash sh_script/integration_tdx.sh -c 2 -m 4G -f target/release/final-boot-kernel.bin
+          bash sh_script/integration_tdx.sh -c 4 -m 8G -f target/release/final-boot-kernel.bin
+          bash sh_script/integration_tdx.sh -c 8 -m 16G -f target/release/final-boot-kernel.bin
```

Comment:
This does make me wonder if we could use a `for` loop in these `run` blocks to avoid the duplication.